elementary/monitor/dbt_project/macros/get_invocations_by_ids.sql

```diff
@@ -0,0 +1,19 @@
+{% macro get_invocations_by_ids(ids) %}
+  {% set database, schema = elementary.target_database(), target.schema %}
+  {% set invocations_relation = adapter.get_relation(database, schema, 'dbt_invocations') %}
+  {% if not invocations_relation %}
+    {% do elementary.edr_log('failed getting invocations relation') %}
+    {% do return(none) %}
+  {% endif %}
+
+  {% set get_invocations_query %}
+    select * from {{ invocations_relation }} where invocation_id in {{ elementary.strings_list_to_tuple(ids) }}
```

Comment:
Lets spread `*` to make it easy to understand which data is returning from this query 🙂